src/main/java/moze_intel/projecte/emc/SimpleStack.java

```diff
@@ -76,7 +82,13 @@ public boolean equals(Object obj)
 			}
 
 			//return this.id == other.id && this.damage == other.damage;
-			return this.id == other.id && this.qnty == other.qnty && this.damage == other.damage;
+			if(this.nbt != null && other.nbt != null){
```

Comment:
May want to ignore NBT for comparison if either are `null` right now, to maintain old behavior.